wcfsetup/install/files/ts/WoltLabSuite/Core/Ui/Notification.ts

```diff
@@ -51,7 +53,7 @@ function hide() {
 /**
  * Displays a notification.
  */
-export function show(message?: string, callback?: Callback, cssClassName?: string): void {
+export function show(message?: string, callback?: Callback | null, cssClassName?: string): void {
```

Comment:
```suggestion
export function show(message: string = "wcf.global.success", callback: Callback | null = null, cssClassName: string = "success"): void {
```

and then remove the handling within the method body.

Reply:
This doesn't work well, because legacy callees might pass `null` instead of `undefined`.